source/material_model/drucker_prager.cc

```diff
@@ -70,7 +70,7 @@ namespace aspect
               // The negative of the second principle invariant is equal to 0.5 e_dot_dev_ij e_dot_dev_ji,
               // where e_dot_dev is the deviatoric strain rate tensor. The square root of this quantity
               // gives the common definition of effective strain rate.
-              const double edot_ii_strict = (&(this->get_simulator()) == NULL
+              const double edot_ii_strict = (this->simulator_is_initialized()
```

Comment:
`simulator_is_initialized()` returns a `bool`. The other function returned a pointer.